tutorials/packaging/create-your-first-snap/create-your-first-snap.md

```diff
@@ -88,29 +94,27 @@ Duration: 4:00
 
 ### Starting the project
 
-The first thing to do is to create a general snaps directory followed by a working directory for
-this specific snap project:
+The first thing to do is to create a general snaps directory followed by a working directory for this specific snap project:
 
 ```bash
-mkdir -p ~/mysnaps/hello
-cd ~/mysnaps/hello
+$ mkdir -p ~/mysnaps/hello
```

Comment:
are we using '$' now?

Reply:
only to remain consistent with the wider snap/snapcraft documentation.